config/source.go

```diff
@@ -58,37 +58,59 @@ func ParseSource(cfg map[string]string) (Source, error) {
 		BatchSize:      defaultBatchSize,
 	}
 
-	keyColumns := strings.Split(cfg[KeyColumns], ",")
-	for i := range keyColumns {
-		if keyColumn := strings.TrimSpace(keyColumns[i]); keyColumn != "" {
+	if cfg[KeyColumns] != "" {
+		keyColumns := strings.Split(cfg[KeyColumns], ",")
```

Comment:
Maybe we can do TrimSpace here. What do you think?
`keyColumns := strings.Split(TrimSpace(cfg[KeyColumns]), ",")`

Reply:
We can, but I don't know why, because we'll be trimming the spaces in the next range.

Reply:
But we have to trim each key after the split.

Reply:
Done.